core/src/main/java/org/geysermc/geyser/util/InventoryUtils.java

```diff
@@ -184,11 +185,22 @@ public static IntFunction<ItemData> createUnusableSpaceBlock(String description)
 
         root.put("display", display.build());
         return protocolVersion -> ItemData.builder()
-                .id(Registries.ITEMS.forVersion(protocolVersion).getStoredItems().barrier().getBedrockId())
+                .id(getUnusableSpaceBlockID(protocolVersion))
                 .count(1)
                 .tag(root.build()).build();
     }
 
+    private static int getUnusableSpaceBlockID(int protocolVersion) {
+        String unusableSpaceBlock = GeyserImpl.getInstance().getConfig().getUnusableSpaceBlock();
+        ItemMapping unusableSpaceBlockID = Registries.ITEMS.forVersion(protocolVersion).getMapping(unusableSpaceBlock);
+        if (unusableSpaceBlockID != null){
```

Comment:
`if (unusableSpaceBlockID != null) {`

Reply:
oh gosh, how'd i miss that
thanks, fixed it!